packages/flutter/test/animation/animation_controller_test.dart

```diff
@@ -350,15 +348,15 @@ void main() {
     expect(controller.repeat, throwsFlutterError);
 
     controller.dispose();
-    FlutterError result;
+    FlutterError? result;
```

Comment:
You can make this `late`.

Reply:
I still think nullable is the way to go. I like the `expect(result, isNotNull);` line; removing it from the test feels off.